src/core/services/compiler/compiler.js

```diff
@@ -6,7 +6,21 @@
  */
 angular
   .module('material.core')
-  .service('$mdCompiler', MdCompilerService);
+  .service('$mdCompiler', MdCompilerService)
+  .provider('$$mdPreAssignBindings', PreAssignBindingsProvider);
+
+/**
+ * Provider that is used to report the preAssignBindingsEnabled state to the $mdCompiler service.
+ */
+function PreAssignBindingsProvider($compileProvider) {
+  // To avoid breaking changes in Material, AngularJS Material will only respect the
+  // preAssignBindingsEnabled state in AngularJS 1.6.X and higher.
+  var respectPreAssignState = angular.version.major === 1 && angular.version.minor > 5;
```

Comment:
This will break with AngularJS `~1.5.10` if the developer sets the `preAssignBindingsEnabled` flag to `false`.

What are the breaking changes mentioned in a comment?